liquibase-standard/src/main/java/liquibase/change/ChangeFactory.java

```diff
@@ -120,6 +134,43 @@ public Change create(String name) {
         }
     }
 
+    /**
+     * Verify if the supports method is implemented in the change if it's not part of the default liquibase changes
+     */
+    private void verifySupportsMethodImplementation(Set<Change> plugins) {
+        if (GlobalConfiguration.SUPPORTS_METHOD_VALIDATION_LEVEL.getCurrentValue().equals(SupportsMethodValidationLevelsEnum.OFF)) {
+            return;
+        }
+        //we only verify supports method if this is not part of the default liquibase changes
+        for (Change plugin : plugins) {
+            String packageName = plugin.getClass().getPackage().getName();
+            if (packageName.startsWith("liquibase.change")) {
+                continue;
+            }
```

Comment:
Do we even want this? Thinking more about this, don't we want all changes to be verified? If so, then I think this method just introduces another way for an extension to accidentally override the supports method and us not notice.